src/Mod/Draft/importDXF.py

```diff
@@ -333,17 +333,18 @@ def locateLayer(wantedLayer, color=None, drawstyle=None):
     """
     # layers is a global variable.
     # It should probably be passed as an argument.
-    wantedLayerName = decodeName(wantedLayer)
```

Comment:
Am I right in concluding that the `decodeName` function can be removed altogether?

Reply:
Yes, it was only needed for Python 2.